src/theme/components/radio.ts

```diff
@@ -20,11 +20,11 @@ const baseStyleControl = {
     color: "ui.white",
     _disabled: {
       color: "ui.gray.medium",
```

Comment:
Should this one also be `"ui.gray.dark"`.

Reply:
Yes.

Reply:
Same here.